concent_api/core/constants.py

```diff
@@ -90,3 +91,6 @@ class VerificationResult(IntEnum):
 
 # Defines a string that is returned from Bankster to Core views to indicate timestamp error case.
 BANKSTER_TIMESTAMP_ERROR = 'timestamp_error'
+
+
+ZERO_SIGNATURE = b'\0' * Message.SIG_LEN
```

Comment:
it is used only in the tests - it shouldn't be here... Please move it to a new modul `tests_constants` or `constants_for_tests`

Reply:
Done